projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -1773,14 +1773,18 @@ s_tacacs
    (
       t_null
       | t_server
+      (
```

Comment:
Please move this parenthetical expression to the end of t_server definition. It is the handler for t_server that would use information in this expression.